x/incentives/client/cli/flags.go

```diff
@@ -0,0 +1,24 @@
+package cli
+
+import (
+	flag "github.com/spf13/pflag"
+)
+
+// flags for lockup module tx commands
+const (
+	FlagLockQueryType = "lock_query_type"
+	FlagDenom         = "denom"
+	FlagDuration      = "duration"
+	FlagTimestamp     = "timestamp"
+)
+
+// FlagSetCreatePot returns flags for creating pot
+func FlagSetCreatePot() *flag.FlagSet {
+	fs := flag.NewFlagSet("", flag.ContinueOnError)
+
+	fs.String(FlagLockQueryType, "ByDuration", "ByDuration | ByTime")
+	fs.String(FlagDenom, "stake", "locked denom to be queried")
+	fs.String(FlagDuration, "86400s", "The duration token to be locked. e.g. 1h, 1m, 1s, 0.1s")
```

Comment:
Can we make the default `1w`?

Reply:
golang natively support until hour maximum unit. I think it's CLI stuff and we could introduce extra package which support days and weeks as well. What packages do you recommend, what I have been using are not popular pacakges.

Reply:
Oh huh, I had no idea about this. Relevant issue if anyones interested: https://github.com/golang/go/issues/11473

Could we use https://pkg.go.dev/github.com/rickb777/date/period, and just have the CLI add the `P` prefix, and capitalization itself?